deploy.yml

```
@@ -0,0 +1,36 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:

```

```suggestion
jobs:
   check-linting:
    name: Check linting
    needs: build-data
    permissions:
      contents: read
    uses: ./.github/workflows/lint.yml

  deploy:
    runs-on: ubuntu-latest
    needs: [check-linting]
    steps:
```
Para asegurar que antes del desplegado se haya revisado el linting y formattig